pkg/services/app/add.go

```diff
@@ -673,24 +672,19 @@ func (a *App) createPullRequestToRepo(params AddParams, basePath string, repo st
 
 	owner, err := getOwnerFromUrl(repo)
 	if err != nil {
-		return nil
```

Comment:
:cold_sweat: 

I'm trying to think of a way to catch this with linting...

Reply:
Boy, that would be nice!